bin/barnard59.js

```diff
@@ -64,6 +67,19 @@ const onError = async err => {
     const spanProcessor = new BatchSpanProcessor(exporter)
     sdk.configureTracerProvider({}, spanProcessor)
   }
+  // Export the metrics to a collector. By default it exports to
+  // http://localhost:55681/v1/metrics, but it can be changed with the
+  // OTEL_EXPORTER_OTLP_METRICS_ENDPOINT environment variable.
+  if (otelMetricsExporter === 'otlp') {
+    const exporter = new CollectorMetricExporter()
+    sdk.configureMeterProvider({
+      exporter
```

Comment:
```suggestion
      exporter,
      interval: 10000, // Export metrics every 10 seconds
```

Reply:
Oh, I would not like to hardcode that. Override with env var maybe?

Reply:
Or an additional `--otel-metrics-interval <ms>` option